lib/galaxy_test/selenium/test_pages.py

```diff
@@ -19,14 +23,41 @@ def test_simple_page_creation_edit_and_view(self):
         self.screenshot("pages_grid")
         name = self.create_page_and_edit(screenshot_name="pages_create_form")
         self.screenshot("pages_editor_new")
-        self.components.pages.editor.markdown_editor.wait_for_and_send_keys("moo\n\n\ncow\n\n")
-        self.components.pages.editor.embed_dataset.wait_for_and_click()
+        editor = self._page_editor
+        editor.markdown_editor.wait_for_and_send_keys("moo\n\n\ncow\n\n")
+        editor.embed_dataset.wait_for_and_click()
         self.screenshot("pages_editor_embed_dataset_dialog")
-        self.components.pages.editor.dataset_selector.wait_for_and_click()
+        editor.dataset_selector.wait_for_and_click()
         self.sleep_for(self.wait_types.UX_RENDER)
-        self.components.pages.editor.save.wait_for_and_click()
+        editor.save.wait_for_and_click()
         self.screenshot("pages_editor_saved")
-        self.home()
+        self.page_open_with_name(name, "page_view_with_embedded_dataset")
+
+    @selenium_test
+    @managed_history
+    def test_workflow_problem_display(self):
+        workflow_populator = self.workflow_populator
+        problem_workflow_1_id = workflow_populator.upload_yaml_workflow(
```

Comment:
IMO we shouldn't even go to the depth of checking tool versions (ok, maybe it is necessary?) and runtime state (I'm more confident that isn't a good idea) when embedding the workflow in the page ... if you've created a page you wouldn't want it to start showing errors or warnings that only concern you if you're trying to re-run that analysis.

See https://github.com/galaxyproject/galaxy/pull/15807#discussion_r1138342637 ... I think not doing runtime inputs would prevent (at least some of) these errors popping up in the workflow preview.